hypha/settings/base.py

```diff
@@ -524,6 +525,13 @@
 SLACK_DESTINATION_ROOM_COMMENTS = env.str('SLACK_DESTINATION_ROOM_COMMENTS', None)
 SLACK_TYPE_COMMENTS = env.list('SLACK_TYPE_COMMENTS', [])
 
+# Django Slack settings
+SLACK_TOKEN = env.str('SLACK_TOKEN', None)
+SLACK_USERNAME = env.str('SLACK_USERNAME', 'bot')
+SLACK_BACKEND = 'django_slack.backends.CeleryBackend'  # UrllibBackend can be used for sync
+if SLACK_DESTINATION_URL:
+    SLACK_ENDPOINT_URL = SLACK_DESTINATION_URL
```

Comment:
@sandeepsajan0, let's get `SLACK_ENDPOINT_URL` directly from env. So replace `SLACK_DESTINATION_URL = env.str('SLACK_DESTINATION_URL', None)` with `SLACK_ENDPOINT_URL = env.str('SLACK_ENDPOINT_URL', None)`, and change `SLACK_DESTINATION_URL` to `SLACK_ENDPOINT_URL` everywhere.

We will have to set `SLACK_TOKEN` anyway for this to work so we can also change `SLACK_DESTINATION_URL` to `SLACK_ENDPOINT_URL` at the env settings.

Reply:
This avoids confusion for someone who is looking at this in future.

Reply:
okay, I was thinking of not changing the existing env variables. But yes, we will have to add SLACK_TOKEN so renaming should not be a big issue. I have changed it in the latest commit.